src/cloud_utils/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.10-bullseye AS build
```

Comment:
Python 3.11. All our ugbio packages and dockers are 3.11. This is needed for the pre-commit to work